### ISSUE TITLE
a way to reread settings for pylance on virtual environment changes

### ISSUE THREAD

**Issue by fenchu** [microsoft/pylance-release#544]
os - win10 2004
pylance: v2020.10.3
vsc: 1.50.1

I've not found any ways to reread my environment when I run pip in console to change modules, when I recreate virtual environment or change python version.

selecting python interpreter or changing it does not trigger any update.
I may be it rereads after some long time, but I havent had time to wait.

I have to close and reopen project for pylance to reread my virtual environment. and then it is all fine.

The output is all red underlined with inaccessible modules which has been changed in current virtual environment.

Nice to get a shortcut to reread for current virtual environment, or pressing the python interpreter status will reread it.

This is a great product and a massive improvement upon the old Python language server.

Thanks

**Comment by jakebailey** [comment]
We should already support pip installs via file watching, and the Python extension notifies us when the interpreter changes.

The core extension recently updated; was this working previously or has this only been broken for you for a short time?

**Comment by fenchu** [comment]
Hi
I think I found the error, due to using Darkula Python skin, which has a binding to MagicPython, I removed both, and now things seem to work fine.

I do not have windows at home, but I had the same error here with same vsc + plugins on linux mint. and now changes to pip are reflected in vsc immediately.

I also use both python.pythonPath and python.analysis.extraPaths pointing to the same pyenv environment:
```
{
    "python.pythonPath": "/home/mortenb/.pyenv/versions/pygame/bin/python",
    "python.analysis.extraPaths": ["~/.pyenv/versions/pygame/bin/python"]
}
```
